### ISSUE TITLE
[ENH] Logo for FastF1

### ISSUE THREAD

**Issue by theOehrly** [theOehrly/Fast-F1#365]
### Proposed new feature or change:

There are a few places where it would be nice to have a logo for FastF1. I'm looking for suggestions and proposals for a logo. There are a few requirements:

- simple and modern design
- useable in the readme and documentation but also useable in the form of the common square/round user avatar
- should preferably not contain the letters "F1" (for legal reasons, among others)
- should not use the official F1 font (legal reasons again)

**Comment by pesaventofilippo** [comment]
Definitely not a suggestion to use this logo as it's very basic, but I think something like this would be nice.
A simple logo (with text or not) that in some way represents stats/telemetry.

![logo](https://github.com/theOehrly/Fast-F1/assets/32228386/f73f2fe6-99a2-468c-b422-c53180c9053d)

I made this with matplotlib and the Formula 1 font, so both are easily adjustable

**Comment by theOehrly** [comment]
@pesaventofilippo yes, kind of goes in the right direction I think. Conveying stats and telemetry is good, somehow incorporating makes it difficult, I think. You chose team colors which I think is a cool idea.

Hopefully some more people can weigh in with ideas and opinions.

But also, we can't use the official Formula 1 font, for the same reasons that we shouldn't use the letters "F1". A bit ridiculous, but not worth taking any risks there.

**Comment by pesaventofilippo** [comment]
Yeah I agree on the F1 font. Besides that, if you find a more "detailed" logo (as in, with more stuff going on and not as simple as mine), I don't think it needs text at all. If you can't write the library name (even FF1 still contains 'f1'), there's not much that makes sense to write.

Maybe instead of a simple bar plot it could be a stylized line chart of some sort? (but then you'd risk making it too complex and difficult to recognize when viewed as a small image, so I don't know)

**Comment by Casper-Guo** [comment]
This issue can really benefit from much better visibility. With permission, I think it will be good to shout it out on Twitter, Reddit etc. and hopefully get some community submissions

**Comment by theOehrly** [comment]
Sure, feel free to do that, if you like. I don't have a noteworthy social media presence at the moment, so I can't really do that. (I've reserved some relevant usernames, but those accounts are not active.)
On the other hand, this issue is not really very important. It's very much a "nice to have" thing. But it'd be cool if there was a logo, of course.

**Comment by MooshuBeef** [comment]
> useable in the readme and documentation but also useable in the form of the common square/round user avatar

Are .png or .jpg file for this? Gonna brainstorm a bit, design is more my forte than it is my coding that's for sure 😆

**Comment by theOehrly** [comment]
> > useable in the readme and documentation but also useable in the form of the common square/round user avatar
> 
> Are .png or .jpg file for this? Gonna brainstorm a bit, design is more my forte than it is my coding that's for sure 😆 

Yes, I was referring to shape/aspect-ratio and the size of details here.

**Comment by RickyRAV** [comment]
It ain't much, but maybe it can help someone who's better.
![FastF1Logo](https://github.com/theOehrly/Fast-F1/assets/101335214/1b5db608-3e5d-4abf-a545-65777ffa35c1)

**Comment by theOehrly** [comment]
@RickyRAV I kind of like the theme 👍 I guess it's more of a design concept and not really a logo at the moment. But maybe someone is inspired to create someting based on it.

**Comment by Ax6** [comment]
Inspired from @pesaventofilippo, here's my submission 

![FASTF1Logo](https://github.com/theOehrly/Fast-F1/assets/6977926/dfec1c2a-2834-4206-b1f9-b99009d73c95)

**Comment by theOehrly** [comment]
@Ax6 I like that logo. It is pretty recognizable and unique. I'm wondering if it might be a good idea to just leave out the text?
That would make it a bit less "full of lines". And it would remove the "F1" trademark. I'm not sure if that legal stuff might actually become an issue, though.

**Comment by theOehrly** [comment]
![grafik](https://github.com/theOehrly/Fast-F1/assets/23384863/aade13b0-0b92-442f-a13f-854a6bcf489f)

Here is another suggestion from me. I felt like doing some non-coding stuff for a change 😅
I drew a lot of inspiration from the already suggested logos and tried to combine some elements while keeping it simple enough.
Shown are an avatar version and a banner version. I'd use the logo with a transparent background, so it works on a dark and white background. Opinions?

**Comment by pesaventofilippo** [comment]
I like it! It's simple enough, yet represents FastF1 very well IMO. The car is a very nice touch as well.
In the end are you planning to keep "F1" in the logo? (I guess so, at least it makes sense if the package is named that way also).
If I had to make a suggestion, I'd make the first column a slightly darker green, because it doesn't "pop" as much as the red/orange with a white background. Unless it's referring to the Merc team color, then changing it doesn't make sense.

**Comment by theOehrly** [comment]
Yes, I decided that we can keep the "F1" in the text in this form. It's integrated into the name, not a separate word, and a very different font from the official logo. It's just the name of the project, and so far that hasn't been a problem. If they decide at some point that this isn't OK, then the logo will be a minor problem, because I'd probably have to rename the whole project and changing the name in that logo is quickly done.

Thanks for the suggestion with the colour. I'll play around with that a bit more. I kind of like the slightly muted colours, but to be fair, I haven't even tested other colours for the bars. And you are right, it doesn't stand out that much.
